pkg/state/history.go

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package state
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/lib/pq"
+	"github.com/xataio/pgroll/pkg/migrations"
+)
+
+// Migration represents a single migration in the migration history
+// of a schema
+type Migration struct {
+	Migration migrations.Migration
+	CreatedAt time.Time
+}
+
+// SchemaHistory returns all migrations applied to a schema in ascending
+// timestamp order
+func (s *State) SchemaHistory(ctx context.Context, schema string) ([]Migration, error) {
+	rows, err := s.pgConn.QueryContext(ctx,
+		fmt.Sprintf("SELECT name, migration, created_at "+
```

Comment:
nit: You can use backticks to quote the string and include newlines so that you don't need to break it up with `+`

Reply:
done